includes/tacopie/network/io_service.hpp

```diff
@@ -50,13 +50,16 @@ namespace tacopie {
 class io_service {
 public:
   //! ctor & dtor
-  io_service(void);
+  io_service(std::size_t nb_threads = __TACOPIE_IO_SERVICE_NB_WORKERS, bool bDelayedStart = false);
```

Comment:
Seems like bDelayedStart can be removed, not used anywhere, no access from cpp_redis